app/src/main/java/org/apache/taverna/mobile/data/local/DBHelper.java

```diff
@@ -184,6 +186,24 @@ public boolean updateFavouriteWorkflow(String id) {
         return false;
     }
 
+    public Observable<List<Workflow>> getFavouriteWorkflow() {
+        return Observable.create(new Observable.OnSubscribe<List<Workflow>>() {
+            @Override
+            public void call(Subscriber<? super List<Workflow>> subscriber) {
+                if (subscriber.isUnsubscribed()) return;
```

Comment:
This is necessary if we use create operator. 
For further info please refer issue [#12 ](https://github.com/ribot/ribot-app-android/issues/12) at ribot android app